code/controllers/subsystem/job.dm

```diff
@@ -922,15 +922,20 @@ SUBSYSTEM_DEF(job)
 		return JOB_UNAVAILABLE_SPECIES
 
 	if(CONFIG_GET(flag/min_flavor_text))
-		if(length_char(player.client.prefs.read_preference(/datum/preference/text/flavor_text)) <= CONFIG_GET(number/flavor_text_character_requirement))
+		//BUBBER EDIT ADDITION: SILICON FLAVOR TEXT CHECK
+		var/is_silicon = (possible_job.auto_deadmin_role_flags & DEADMIN_POSITION_SILICON)
```

Comment:
This is because this is the latejoin menu, and the player has not chosen a job yet, this is checking their eligibility for the job

I will try this regardless, what variable do you believe A should be?

Reply:
Update: It did not work but I will make the is_silicon variable into a helper